guides/03_features/03_padrino-mailer.html.md

```diff
@@ -102,7 +110,7 @@ MyAppName.mailer :sample do
     locals :name => name, :email => email
     render 'sample/registration_email'
     content_type :html # optional, defaults to :plain
-    via :sendmail      # optional, to smtp if defined otherwise sendmail
+    via :sendmail, settings     # optional, to smtp if defined otherwise sendmail
```

Comment:
> I feel like instead of settings, we should provide a real value here to prevent confusion.

Eh, Padrino attempts to set `:location` by default, but okay, updated...

> ... expand the Configuration section above to include possible settings and where in the Mail gem the documentation can be found.

My commit already contains such a link. Unless have a better doc section of the Mail docs to link to?